defaults/main.yml

```diff
@@ -5,3 +5,12 @@ package_state: "present"
 # default vars for the monitoring agent
 monitoring_package_name: stackdriver-agent
 monitoring_service_name: stackdriver-agent
+
```

Comment:
Yes, I think it makes more sense to do it that way. Can you remind me what `force_upgrade` does again?

Reply:
Amended https://github.com/GoogleCloudPlatform/stackdriver-ansible-role/pull/39/files to account for this change.

Reply:
It sounds like the variables in `main.yml` are all customizable by the users (correct me if that's wrong).

Should we remove the following fields from the `main.yml` then? We do not want users to customize them, right? If users customize them, things may stop working because the underlying package does not support the customized paths / names.

monitoring_package_name
monitoring_service_name
logging_package_name
logging_service_name
logging_binary

Reply:
Thats correct, I think in general users are somewhat discouraged from modifying variables under `/default` whereas variables under`/vars` users are free to modify. However, if we never expect a user to change the binary path, or package names I see no problem in hard coding them.

Also I think it makes sense to move the rest of the vars to the `/vars` folder.